tutorials/governance/opengov/create-a-referenda.md

```diff
@@ -0,0 +1,91 @@
+        - **beneficiary** - destination account for the transfer
+    4. Copy the preimage hash -
+    the preimage hash for this example is `0x0e634ee7f6adf6e7cfb7e2fa6c77454c3d0ce46f64287df0c1da8d079d9633a2`
+    5. Click the **+ Submit preimage** button
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-3.webp)
+
+## Submit a Proposal
+
+Submitting a proposal requires you to bond a specified amount of tokens. The bond value depends on the configuration settings of the chain you use.
+
+Before submitting the proposal, you must first provide the preimage hash. The preimage hash is a cryptographic hash of the full proposal details you intend to enact.
+
+To submit a proposal, follow these steps:
+
+1. Navigate to the **Governance** dropdown and select **Referenda**
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-4.webp)
+2. Click in the **+ Submit proposal** button
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-5.webp)
+3. Fill in the parameters and submit it:
+    1. Select the account from which the proposal will be registered. The balance lock will be applied to this account
+    2. Choose your proposal's origin (and, by extension, the track). Each track has different durations, privileges, and acceptance criteria
+    3. Enter the preimage hash obtained in previous steps. The preimage length field will be automatically populated
+    4. Select the enactment delay. This can be specified either as a block number or as a specific number of blocks after the referendum is approved
+    5. Click the **Submit proposal** button
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-6.webp)
+4. After submitting, you should see a `referenda.Submitted` event with your proposal in the explorer section. The proposal number will be displayed in the **index** field
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-7.webp)
+
+## Submit a Referendum on the Whitelisted Caller Track
+
+Operations deemed safe or time-sensitive by the Polkadot Technical Fellowship can be processed through the Whitelisted Caller track. This specialized track allows proposals to pass more quickly by requiring less voter turnout during the early stages of the decision period. It is primarily used for technical, neutral proposals, such as runtime upgrades or adjustments to the system’s parachain validation configuration. However, unlike other tracks, the Whitelisted Caller track requires a different approach. Instead of voting directly on the proposal, the vote is to dispatch the proposal via the Whitelist pallet.
+
+Before submitting a referendum on this track, it’s crucial to secure a positive signal from the Fellowship that they will whitelist the proposal. Without this endorsement, it won’t be executed even if the referendum passes.
```

Comment:
```suggestion
Before submitting a referendum on this track, it is crucial to secure a positive signal from the Fellowship so that they can whitelist the proposal. Even if the referendum passes, it won’t be executed without this endorsement.
```